webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: 'development',
+  entry: './amt/site/static/ts/amt.ts',
+  output: {
+    filename: 'amt.js',
+    path: path.resolve(__dirname, 'amt/site/static/dist'),
+    publicPath: 'amt/site/static/dist'
```

Comment:
This path needs a trailing / for paths to resolve correctly:
```suggestion
    publicPath: 'amt/site/static/dist/'
```